### ISSUE TITLE
Visualise the point density on the building (potential student work)

### ISSUE THREAD

**Issue by balazsdukai** [tudelft3d/3dbag-viewer#72]
It would be neat if the point density could be visualised on the surface of the building models, to indicate how much data support is there in the point cloud. I'm thinking about a heatmap as a texture, computed and draped over the whole building. I'm assuming that a sufficiently low-resolution texture would be more efficient to visualise than pulling in the raw point cloud for instance. But who knows?

Basically, the goal is to show the user which surfaces very low amount or zero points. More or less. This could help people better understand what is going on with the reconstruction if some parts look funny.

Ideas are welcome. :-)
